mods/cnc/maps/nod02a/rules.yaml

```diff
@@ -5,7 +5,7 @@ World:
 		StartingMusic: ind2
 		VictoryMusic: nod_win1
 	MissionData:
-		Briefing: GDI has kept a stranglehold on Egypt for many years.  Set up a forward attack base in your area.  To do this you must select your Mobile Construction Vehicle (MCV) and right click on it.  From here you can begin to build a base. This area contains plenty of Tiberium, so establishing the base should be easy.
+		Briefing: GDI has kept a stranglehold on Egypt for many years. Establish a forward attack base in the region. Begin by selecting your Mobile Construction Vehicle (MCV) and right-clicking on it to start base construction. This area contains plenty of Tiberium, so establishing a base should be straightforward.
```

Comment:
Change start to enable. Otherwise this implies that the base will build itself. Likewise for Nod02b.